### PR TITLE
feat: add productTemplateV2

### PR DESCRIPTION
**Pull request by admmasters**

### CODE REVIEW

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+// THEME ME ProductTemplate component
```
Comment (admmasters) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:1]: We should have a themable ProductTemplate

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    height: string;
+  };
+}
+type MaybeHeaderProps = {
+  title?: React.ReactNode;
+  subtitle?: React.ReactNode;
+  progressBar?: React.ReactNode;
+};
+
+const MaybeHeader = ({ title, subtitle, progressBar }: MaybeHeaderProps) => (
```
Comment (zopa-charlotte-walker) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:24]: why maybe header?

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  backgroundColor: string;
+  header: {
+    backgroundColor: string;
+    height: string;
+  };
+}
+type MaybeHeaderProps = {
+  title?: React.ReactNode;
+  subtitle?: React.ReactNode;
+  progressBar?: React.ReactNode;
```
Comment (OlenaKashuba) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:21]: I like that we can pass a component inside, exactly what we need

File: src/index.ts [hunk 4]
```diff
@@ -71,6 +71,8 @@ export * from './components/organisms/Tabs';
 export * from './components/organisms/Carousel';
 // Templates
 export { default as ProductTemplate } from './components/templates/ProductTemplate';
+export { ProductTemplate as ProductTemplateV2 } from './components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2';
```
Comment (OlenaKashuba) [inline src/index.ts:74]: do you have an example of usage of v2 to see how different they are?

File: src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+};
+
+const MaybeHeader = ({ title, subtitle, progressBar }: MaybeHeaderProps) => (
+  <header className="zrc:header">
```
Comment (zopa-charlotte-walker) [inline src/components/templates/ProductTemplate/ProductTemplate/ProductTemplateV2.tsx:25]: is this zrc:header a pattern we usually use? not seen before

File: src/index.ts [hunk 6]
```diff
@@ -71,6 +71,8 @@ export * from './components/organisms/Tabs';
 export * from './components/organisms/Carousel';
 // Templates
```
Comment (zopa-charlotte-walker) [inline src/index.ts:72]: do we need these comments

**Review by zopa-charlotte-walker — APPROVED** [review]

**Pull request merged**